packages/insomnia-app/app/ui/components/settings/general.js

```diff
@@ -326,10 +332,34 @@ class General extends React.PureComponent<Props, State> {
         </div>
 
         <div className="form-row">
+          <div className="form-control form-control--outlined">
+            <label>
+              Editor Font Weight
+              {fontsMono ? (
+                <select
+                  name="fontMonospaceWeight"
+                  value={settings.fontMonospaceWeight}
+                  onChange={this._handleFontWeightChange}>
+                  {EDITOR_FONT_WEITGHTS.map((item, index) => (
+                    <option key={index} value={item.value}>
+                      {item.label}
+                    </option>
+                  ))}
```

Comment:
It's good to avoid using array index as keys in React. While it's not as concerning in this particular use-case, it is still an [anti-pattern](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318).

After converting `EDITOR_FONT_WEITGHTS` to an object, you can use `Object.entries(EDITOR_FONT_WEITGHTS)` to get the key-value pairs.

```suggestion
                  {Object.entries(EDITOR_FONT_WEITGHTS).map(e => (
                    <option key={e[1]} value={e[1]}>
                      {e[0]}
                    </option>
                  ))}
```